contributing/community/releases.rst

```diff
@@ -1,36 +1,34 @@
 The Release Process
 ===================
 
-This document explains the Symfony release process (Symfony being the code
-hosted on the main ``symfony/symfony`` `Git repository`_).
+This document explains the **release process** of the Symfony project (i.e. the
+code hosted on the main ``symfony/symfony`` `Git repository`_).
 
-Symfony manages its releases through a *time-based model*; a new Symfony minor
-version comes out every *six months*: one in *May* and one in *November*.
+Symfony manages its releases through a *time-based model* and follows the
+`Semantic Versioning`_ strategy:
 
-.. tip::
-
-    The meaning of "minor" comes from the `Semantic Versioning`_ strategy.
-
-Each minor version sticks to the same very well-defined process where we start
-with a development period, followed by a maintenance period.
+* A new Symfony minor version (e.g. 2.8, 3.2, 4.1) comes out every *six months*:
+  one in *May* and one in *November*;
+* A new Symfony major version (e.g., 3.0, 4.0) comes out every *two years* and
+  it's released at the same time of the last minor version of the previous branch.
 
 .. note::
 
-    This release process has been adopted as of Symfony 2.2, and all the
-    "rules" explained in this document must be strictly followed as of Symfony
-    2.4.
+    This release process has been adopted as of Symfony 2.2, and all the "rules"
+    explained in this document must be strictly followed as of Symfony 2.4.
```

Comment:
I propose to remove this note entirely. It made sense when we did the transition to the new formal process. But that was years ago!